doc/grpc_xds_features.md

```diff
@@ -54,6 +54,7 @@ xDS features.
 Features | gRFCs  | [C++, Python,<br> Ruby, PHP](https://github.com/grpc/grpc/releases) | [Java](https://github.com/grpc/grpc-java/releases) | [Go](https://github.com/grpc/grpc-go/releases) | [Node](https://github.com/grpc/grpc-node/releases)
 ---------|--------|--------------|------|------|------
 **xDS Infrastructure in gRPC client channel:**<ul><li>LDS->RDS->CDS->EDS flow</li><li>ADS stream</li></ul> | [A27](https://github.com/grpc/proposal/blob/master/A27-xds-global-load-balancing.md) | v1.30.0  | v1.30.0 | v1.30.0 | v1.2.0 |
+**xDS v2 support:** | [A27](https://github.com/grpc/proposal/blob/master/A30-xds-v3.md#details-of-the-v2-to-v3-transition) | < v1.51.0  | < v1.53.0 | TBA | < v1.8.0 |
```

Comment:
Please move this down to be right under the row for xDS v3 support.

Also, no need to make this bold, and no need for the trailing `:` character.

Reply:
Good point, thanks. Didn't notice the v3 somehow.